### ISSUE TITLE
Python 3.12 No support 

### ISSUE THREAD

**Issue by Eranh0** [Abjad/abjad#1574]
should work with python 3.10 and greater, but doesn't work with 3.12

`pip3 install abjad
Defaulting to user installation because normal site-packages is not writeable
Collecting abjad
  Using cached abjad-3.19.tar.gz (413 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [3 lines of output]
      This is Abjad 3.19, which supports Python 3.10, 3.11.

      But it looks like you're trying to install Abjad with Python 3.12.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.`

**Comment by trevorbaca** [comment]
That's right: Abjad 3.19 has only been tested with Python 3.10 and 3.11.

I'll test with Python 3.12 when I build Abjad 3.20.

**Comment by ghkarl7777** [comment]
It's been a year.  Python 3.13 is out.  Will that be supported soon?

**Comment by trevorbaca** [comment]
@ghkarl7777 Yes, and until then remember that you can use a virtual environment with the versions of Python that are supported.

**Comment by trevorbaca** [comment]
Merged in #1600
